Ibd partitioning

RosCraddock opened the pull request:
Code completed for implementing gametic/ibd partitioning in R.
Tested on one and multiple traits locally. Implemented test for single and multiple traits.
All other testing passed through test_that.

Addressing issue #18

RosCraddock committed:
Merge pull request #3 from AlphaGenes/devel

Devel

RosCraddock committed:
IBD partitioning & other fixes

-IBD partition
-Fixed missing g in alphapartdropgroup()
-removed centre=FALSE

RosCraddock committed:
Update documentation

RosCraddock committed:
ibd corrections

- column names
- handling multiple traits

RosCraddock committed:
Merge branch 'ibd' of https://github.com/RosCraddock/AlphaPart into ibd

RosCraddock committed:
More corrections

- multiple traits with IBD partitioning

RosCraddock committed:
completed to do

RosCraddock committed:
Test for IBD

- test for IBD (single trait)
- commenting break downs the theory

RosCraddock committed:
Test for multiple traits

RosCraddock committed:
minor

gregorgorjanc commented:
This looks very impressive @RosCraddock !!!!! I have to finish a couple of other bits, so will lag with the review.

RosCraddock committed:
ibd vignette and minor update

- First draft of ibd vignette to demonstrate ibd partitioning in AlphaPart.
- minor update to c++ code.

RosCraddock committed:
Update to error chandling

Small update in error handling for ibd. Otherwise, an error would be flagged unnecessarily. Tested and working on real data.

RosCraddock committed:
minor changes

RosCraddock committed:
Small change
